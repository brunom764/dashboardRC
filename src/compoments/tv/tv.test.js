import { render } from "@testing-library/react"

import TV from "./index"

describe("<TV/>", () => {
  it("should match snapshot", () => {
    const { container } = render(<TV />)

    expect(container).toMatchSnapshot()
  })
})





