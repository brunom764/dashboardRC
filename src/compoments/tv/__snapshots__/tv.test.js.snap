// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TV/> should match snapshot 1`] = `
<div>
  <section
    class="px-5 pt-15 pb-10"
  >
    <h1
      class="title text-center pt-5 pb-1 text-green-700"
    >
      Tv robôCIn
    </h1>
    <h4
      class="text-center text-gray-600 p-4"
    >
      Nos acompanhe pelo YouTube e conheça mais do nosso projeto
    </h4>
    <div
      class="flex flex-col md:flex-row justify-around p-2"
    >
      <div
        class="mb-5 mt-5 pb-2 md:w-1/3 shadow-xl"
      >
        <div
          style="width: 100%; height: 40vh;"
        />
        <h2
          class="subtitle text-center pt-2"
        >
          Ep. 1 | Mini Doc. RobôCIn
        </h2>
      </div>
      <div
        class="mb-5 mt-5 pb-2 md:w-1/3 shadow-xl"
      >
        <div
          style="width: 100%; height: 40vh;"
        />
        <h2
          class="subtitle text-center pt-2"
        >
          Ep. 2 | Mini Doc. RobôCIn
        </h2>
      </div>
    </div>
  </section>
</div>
`;
